### DISCUSSION TITLE
Edge Stable, W10, Ublock, Cloudflare

### DISCUSSION THREAD

**Discussion by IgnoreBox** [uBlockOrigin/uBlock-issues#2390] [General]
hello.

i dont know is this my error only or... ublockorigin make me cannot login to my cloudflare dashboard?

i always set clear cookies cache hostory on close browser. so everytime i want to enter cloudflare dashboard. it will need got o login screen.

Username and password are correct
but with ublockorigin Active = failed login ... keepback to login screen
when i disable ublockorigin.. everything normal

**Comment by mapx-** [comment]
1.
- go into dashboard, support tab, post here `Troubleshooting Information`

2.
Reset uBO, don't modify any uBO setting, update all the filter lists, don't add any other list, disable any other extension / addon in your browser, disable builtin blockers /shields/ tracking protection in brave/ opera / vivaldi/ firefox etc, disable (if any) the web shields in your antivirus / antimalware software / other blocking software, restart the system, reopen the browser, test again.

**Reply by IgnoreBox** [reply]
solved
after remove addon from chrome
back to ublock origin at edge store ...
all normal again..

Thankyou
